packages/react-lowcode/src/codegen/interfaces.ts

```diff
@@ -1,15 +1,20 @@
+import ts, { factory } from "typescript";
 import { SourceLineCol } from "../ast";
+import { HookImport } from "../ast/hooks";
+import { TagImport } from "../ast/tags";
 import { UiFramework } from "./definition/context-types";
 
 export interface CodegenOptions {
     // whitelisted entity names
-    readonly names: string[]
+    readonly names: string[],
+    //template for generating list page
+    pageListTemplate: string,
     // default is MaterialUI
     uiFramework?: UiFramework
 }
 
 export interface InsertOptions {
-    entity: string
+    entityName: string
```

Comment:
Let's have backward compatible API (we are close to a milestone...). Let's have safe time for refactoring...